### ISSUE TITLE
I'm stuck loading model

### ISSUE THREAD

**Issue by Motorr23** [ohmplatform/FreedomGPT#80]
I'm stuck loading model
![Capture d’écran (12)](https://github.com/ohmplatform/FreedomGPT/assets/126995751/772cbbde-5e43-4890-8a96-ca34d0cfe9fa)

**Comment by TobiCooki** [comment]
I was having the exact same issue, what worked for me was double checking the models had correctly download, and by placing the downloaded models on my desktop and using the models from there, with the file system for freedom-gpt still in my program files. A full uninstall (using "add or remove programs" in settings if available) and reinstall using the windows installer on the website rather than installing through git  seemed to help as well.

tldr double check the models got succesfully downloaded and are being used properly. And fully reinstall the program.

**Comment by Motorr23** [comment]
Thanks for aswer me, but nothing change

**Comment by benbois** [comment]
Same issue on linux Ubuntu.. stuck on loading model.
Nothing seems to work!

The .Bin files are still in place after a relaunch the app.

**Comment by Postconceptlab** [comment]
same issue on unbuntu as benbois ubuntu

**Comment by letsgitcracking** [comment]
> Same issue on linux Ubuntu.. stuck on loading model. Nothing seems to work!
> 
> The .Bin files are still in place after a relaunch the app.

Same issue here on Ubuntu

**Comment by leeli0** [comment]
I had this issue on Arch Linux. I found that if you use `yarn start` then you can successfully load the model. In this case, first you need to  **Building the llama.cpp library** according to the **Readme** documentation. Otherwise, you could get an error, like this:

`Failed to start child process: Error: spawn /home/lee/Downloads/freedom-gpt/llama.cpp/main ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:476:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn /home/lee/Downloads/freedom-gpt/llama.cpp/main',
  path: '/home/lee/Downloads/freedom-gpt/llama.cpp/main',
  spawnargs: [
    '-m',
    '/home/lee/Downloads/ggml-model-q4_0.bin',
    '-ins',
    '--ctx_size',
    '2048',
    '-n',
    '-1',
    '-ins',
    '-b',
    '256',
    '--top_k',
    '10000',
    '--temp',
    '0.2',
    '--repeat_penalty',
    '1',
    '-t',
    '7'
  ]
}
`
